ledger/sandbox-on-x/src/test/it/scala/com/daml/ledger/platform/sandbox/auth/RevokeUserRightsAuthIT.scala

```diff
@@ -11,28 +11,31 @@ import com.daml.ledger.api.v1.admin.{user_management_service => ums}
 
 import scala.concurrent.Future
 
-final class RevokeUserRightsAuthIT extends AdminServiceCallAuthTests with UserManagementAuth {
+final class RevokeUserRightsAuthIT
+    extends AdminOrIDPAdminServiceCallAuthTests
+    with UserManagementAuth
+    with GrantPermissionTest {
 
   override def serviceCallName: String = "UserManagementService#RevokeUserRights"
 
-  private def adminPermission =
-    ums.Right(ums.Right.Kind.ParticipantAdmin(ums.Right.ParticipantAdmin()))
-
-  override def serviceCall(context: ServiceCallContext): Future[Any] =
+  def serviceCallWithGrantPermission(
+      context: ServiceCallContext,
+      permission: ums.Right,
+  ): Future[Any] =
     for {
       response <- createFreshUser(context.token, context.identityProviderId)
       userId = response.user.getOrElse(sys.error("Could not load create a fresh user")).id
       _ <- stub(context.token).grantUserRights(
         GrantUserRightsRequest(
           userId = userId,
-          rights = scala.Seq(adminPermission),
+          rights = scala.Seq(permission),
           identityProviderId = context.identityProviderId,
         )
       )
       _ <- stub(context.token).revokeUserRights(
         RevokeUserRightsRequest(
           userId = userId,
-          rights = scala.Seq(adminPermission),
+          rights = scala.Seq(permission),
```

Comment:
When the IDP right is insufficient, this statement will not be reached. It is a bit of a problem because we are not proving that RevokeUserRights is implemented correctly

Reply:
you're right, i need to grant user right with super admin then which will pass, and then the next call will fail

Reply:
I pushed the fix for this one